odxtools/ecu_variant_matching.py

```diff
@@ -0,0 +1,155 @@
+# SPDX-License-Identifier: MIT
```

Comment:
I noticed this only now (sorry): if possible, the file name should correspond to the class name it contains (i.e., `ecu_variant_matcher.py`)